level-ws.js

```diff
@@ -84,7 +84,15 @@ WriteStream.prototype._write = function write (d, enc, next) {
   }
 
   function push (d) {
-    self._buffer.push(d)
+    self._buffer.push({
```

Comment:
If we push transformed objects directly to `_buffer` there's no need to `.map()` to create a new array.